### PR TITLE
[v12] Distroless doc updates

### PR DESCRIPTION
**Pull request by tcsc**
Documentation updates making "distroless" the default image

### CODE REVIEW

**Review by stevenGravy — CHANGES_REQUESTED** [review]
Make work with existing docker installation https://goteleport.com/docs/management/guides/docker/.  This currently fails

**Comment by ptgott** [comment]
Just to double-check, does this need to be forward ported to `master` (or was there already a PR for `master`?)

**Review by r0mant — APPROVED** [review]
lgtm once Steven's comment about something not working is addressed.

**Review by ptgott — CHANGES_REQUESTED** [review]
I could have sworn I left a review of this earlier, but I'm not seeing it in the PR conversation. In any case, the Markdown tables are formatted incorrectly and do not render as tables. We should make sure this looks okay on the docs dev server before merging.

File: docs/pages/includes/docker-images-oss.mdx [hunk 1]
```diff
@@ -10,9 +10,17 @@ version of Teleport from the three most recent release branches. They are
...
+|`public.ecr.aws/gravitational/teleport-distroless:(=teleport.major_version=)`|The latest version of Teleport Open Source|Yes|No|[Distroless Debian 11](https://github.com/GoogleContainerTools/distroless)|
+|`public.ecr.aws/gravitational/teleport-distroless-debug:(=teleport.major_version=)`|The latest version of Teleport Open Source|Yes|Yes|[Distroless Debian 11](https://github.com/GoogleContainerTools/distroless)|
+|`(=teleport.latest_oss_docker_image=)`|The version specified in the image's tag (i.e. (=teleport.version=))|No|No|[Distroless Debian 11](https://github.com/GoogleContainerTools/distroless)|
+|`(=teleport.latest_oss_debug_docker_image=)`|The version specified in the image's tag (i.e. (=teleport.version=))|No|Yes|[Distroless Debian 11](https://github.com/GoogleContainerTools/distroless)|
 
 For testing, we always recommend that you use the latest released version of Teleport, which is currently `(=teleport.latest_oss_docker_image=)`.
+
+The `*-debug` images are intended for troubleshooting deployments, and are not indended for production use.
```
Comment (ptgott) [inline docs/pages/includes/docker-images-oss.mdx:22]:
```suggestion
The `*-debug` images are intended for troubleshooting deployments, not for production use. These include a shell and BusyBox to help you investigate issues.
```

File: docs/pages/includes/enterprise/docker-images.mdx [hunk 2]
```diff
@@ -4,14 +4,23 @@ automatically rebuilt every night.
...
+| `(=teleport.latest_ent_docker_image=)-fips` | Enterprise FIPS | The version specified in the image's tag (i.e. (=teleport.version=)) | No | Yes | [Ubuntu 20.04](https://hub.docker.com/\_/ubuntu) |
+
+For testing, we always recommend that you use the latest release version of Teleport Enterprise, which is currently `(=teleport.latest_ent_docker_image=)`.
+
+[Ubuntu 20.04](https://hub.docker.com/\_/ubuntu)-based images for non-FIPS 
+Teleport are available from our [Legacy Amazon ECR Public repository](https://gallery.ecr.aws/gravitational/teleport-ent).
 
-For testing, we always recommend that you use the latest release version of Teleport Enterprise, which is currently `(=teleport.latest_ent_docker_image=)`.
\ No newline at end of file
+The `*-debug` images are intended for troubleshooting deployments, and are not indended for production use.
```
Comment (ptgott) [inline docs/pages/includes/enterprise/docker-images.mdx:26]:
```suggestion
The `*-debug` images are intended for troubleshooting deployments, not for production use. These include a shell and BusyBox to help you investigate issues.
```

**Comment by tcsc** [comment]
> I could have sworn I left a review of this earlier

The review you're thinking of was on `master`. This is a backport. Fixed in both.

**Review by ptgott — APPROVED** [review]

**Review by stevenGravy — APPROVED** [review]
lgtm

**Pull request merged**